pkg/cmd/build/authprovider.go

```diff
@@ -84,7 +85,8 @@ func (ap *authProvider) Credentials(_ context.Context, req *auth.CredentialsRequ
 	}
 	ac, err := ap.config.GetAuthConfig(req.Host)
 	if err != nil {
```

Comment:
Should we filter for a specific error or we just ignore any error?

Reply:
I imagine that any error that may occur when obtaining the credential store would be reflected later, as well as the proposed solution for the described error